action.yml

```diff
@@ -1,12 +1,10 @@
 name: 'ROS industrial_ci'
+description: "Running various CI checks for ROS package repositories"
+
 inputs:
   config:
     description: 'Configuration in JSON format, e.g. from matrix'
     required: false
 runs:
-  using: "composite"
-  steps:
-    - run: $GITHUB_ACTION_PATH/.github/action.sh
-      shell: bash
-      env:
-        INPUT_CONFIG: ${{ inputs.config }}
```

Comment:
You lost the `INPUT_CONFIG`, didn't you?

Reply:
No.. it should still work.
If I recall it correctly, for composite actions you have to pass the inputs explicitly.
The tests use config, so I am sure that this works.